docs/view_components/flow_chart.rst

```diff
@@ -0,0 +1,10 @@
+Flow chart of the tool
+----------------------
+
+The flow chart is a recurrent theme in simulation software involving many number of steps in the progression to accomplish the task.
```

Comment:
The description is too vague here.

Reply:
This should have been part of next PR, instead I forgot and started doing it on this one itself. I will stop here and do it in a different PR and implement your suggestions.